docs/guides/admin/docs/changelog.md

```diff
@@ -218,24 +351,14 @@ Opencast 11
   Replace native javascript with angularJS code
 - [[#3127](https://github.com/opencast/opencast/pull/3127)] -
   Fix minor bugs in AdaptivePlaylist
-- [[#3180](https://github.com/opencast/opencast/pull/3180)] -
```

Comment:
Maybe, we shouldn't do this.

Reply:
This file has (traditionally) been auto generated. That being said, you're the rm here :)

Reply:
I've hit those request limits, but mainly when I have to rerun the script. I just wait for an hour or two and it resets :D